frontend/src/routes/console.tsx

```diff
@@ -57,6 +58,11 @@ export const ConsoleRoutes = () => {
         path={ROUTES.Console.NEW_LOGOS_SIDEBAR}
         component={LogosSideBar}
       /> */}
+      <PrivateRoute
+        exact
+        path={ROUTES.Console.NEW_COLORS_EDIT}
```

Comment:
@arjunsinghy96  now you can server this component on the old URL, since now we have a course outline page to show the changes